book/content/remote_collaboration/checklist/checklist.md

```diff
@@ -798,30 +798,58 @@ This checklist also aims to help participants support the event organisers by be
 
 <a name="checklist-for-managing-and-participating-in-a-distributed-team"></a>
 ## Checklist for managing and participating in a distributed team
-*HELP WANTED*
+Managing and participating in a distributed team can be difficult. This is the magic spell for increasing productivity and team happiness for a distributed team. 
 
 <a name="tips-for-all-the-members"></a>
 ### Tips for all the members
-<details>
-  <summary>Click to expand</summary>
-
-</details>
+- [ ] Be courteous and have realistic exceptations from your colleagues  
+- [ ] Complete the tasks assigned to you
+- [ ] Always speak up your doubt to have a clear perspective on the team goals
+- [ ] If not able to attend a corresponding online meeting, send regards prior to the manager
+- [ ] Read meeting minutes prior and after the online meeting
+- [ ] Help fellow struggling colleagues 
+- [ ] Be transparent with ideas, take and share appropriate credits
 
 <a name="specific-tips-for-the-managers"></a>
 ### Specific tips for the managers
-<details>
-  <summary>Click to expand</summary>
+<!---<details>
+  <summary>Click to expand</summary>--->
+
+- [ ] Clarify all Rules of Communication by relying on Right Remote Work Tools according to the demands of the team:
```

Comment:
relying on Right --> relying on the Right

Reply:
I believe there is a section under construction for "Choice of tools and security aspects" -- perhaps it is useful linking to that section (once it is available?)

Reply:
> I believe there is a section under construction for "Choice of tools and security aspects" -- perhaps it is useful linking to that section (once it is available?)

Will link it once it is available.